setup.cfg

```diff
@@ -1,5 +1,6 @@
 [pycodestyle]
 exclude = .eggs,ENV,build,docs/conf.py,venv
+max-line-length = 88
```

Comment:
Just curiosity: this new line length was introduced by black formatting standard?

Reply:
yes, @italovalcy